src/emailalert/email_function.py

```diff
@@ -10,16 +10,21 @@ def lambda_handler(event, context):
     SUBJECT = "Failure alert for RDS Restore Pipeline"
     result = {}
     if 'status' in event:
-        result[constants.ERROR] = event['taskname'] + 'Error'
+        result['failed step'] = event['taskname']
         result[constants.CAUSE] = event['status']
     elif 'Error' in event:
-        result[constants.ERROR] = event['Error']
+        result['database id'] = event['Identifier']
```

Comment:
can we move the initialisation of `result['database id']` and `result['snapshot id']` outside of the if else loop as these will be common